src/component/DisplayRecipe.js

```diff
@@ -0,0 +1,12 @@
+function ShowRecipe({ recipe }) {
```

Comment:
Don't name exported React component differently from the file name.

Reply:
I changed its name in the later commit which has been removed hii